chapters/equations.tex

```diff
@@ -762,6 +762,15 @@ \section{Initialization, initial equation, and initial algorithm}\label{initiali
 \lstinline!when initial() then! or \lstinline!when {$\ldots$, initial(), $\ldots$} then!. In this case, the algorithmic statements within the when-statement remain active during the whole
 initialization phase.
 
+An active when-clause inactivates the following \lstinline!elsewhen! (similarly as for when-clauses during simulation), but apart from that
+the first \lstinline!elsewhen initial() then! or \lstinline!elsewhen {$\ldots$, initial(), $\ldots$} then! is similarly active during initialization as
+\lstinline!when initial() then! or \lstinline!when {$\ldots$, initial(), $\ldots$} then!.
```

Comment:
Oh, maybe this was also included among the intended references of the _see below_ above?

I think we can say this more directly, if we go back to the first sentence:
> The equations of a when-clause are active during initialization, if and only if they are the first clause of the when-equation explicitly enabled with \lstinline!initial()!, …